static/sass/styles.scss

```diff
@@ -26,3 +26,12 @@
 .p-pull-quote .p-pull-quote__quote:last-of-type::after {
   bottom: auto;
 }
+
+.p-side-navigation--raw-html {
```

Comment:
Can you file an issue on Vanilla and add a comment above in the following format:
```scss
// XXX Carlos - 13/08/20: Can be removed once the following issue has been resolved:
// {{ Link to issue}}
```

Reply:
Is the scrollbar not intended? I mean, are you asking me to file an issue because this is a bug? I thought this was intended because the sidebar is sticky.

Reply:
The scrollbar is intended but we don't want any local overrides/changes as this makes maintenance very difficult. Therefore, the vanilla team need to know that there is a request to change it and it might be something we need to change upstream.

Reply:
Done